generators/app/templates/skeleton/rollup.config.js

```diff
@@ -0,0 +1,6 @@
+var buble = require('rollup-plugin-buble');
```

Comment:
why did you considered to keep this file instead of the following?

```json
{
  "tarima": {
    "bundleOptions": {
      "rollup": {
        "plugins": [
          "rollup-plugin-buble"
        ]
      }
    }
  }
}
```